tests/unit/policy_test.py

```diff
@@ -894,109 +894,80 @@ def test_surface_policy_moves_back_to_object(self):
 
                 #  Step | Action           | Motor-only? | Obs processed? | Source
                 # ------|------------------|-------------|----------------|-------------
-                #  1    | MoveForward      | True        | False          | dynamic_call
+                #  0    | None             | True        | False          | first_step
+                #  1    | MoveForward      | True        | False          | touch_object
                 #  2    | OrientHorizontal | True        | False          | dynamic_call
-                #  3    | OrientVertical   | False       | True           | dynamic_call
+                #  3    | OrientVertical   | KeyError    | True           | dynamic_call
```

Comment:
Why is this a key error now?

Reply:
I know this is a bit of a weird edge case but technically, shouldn't it also be True if `"motor_only_step" not in agent_state.keys()` and only be False if agent_state["motor_only_step"] is False? Basically if motor_only_step is not even in the keys then we for sure don't want to process observations in the LM.

Reply:
> if motor_only_step is not even in the keys then we for sure don't want to process observations in the LM

Could you explain more about this assertion? Why must the `motor_only_step` property be defined for the learning module to process anything? I thought we specifically set `motor_only_step` to bypass the learning module. Why would the learning module know this property exists?